poglink/models/rates.py

```diff
@@ -78,11 +78,13 @@ def update_vals(self, vals, raw=False):
 
         self.extras.update(extras)
 
-    def to_dict(self):
+    def to_dict(self, include_extras=True):
         expected, extras = self.get_expected_and_extras(self.__dict__)
 
         output_dict = expected
-        output_dict.update(extras)
+
+        if include_extras:
+            output_dict.update(extras)
```

Comment:
Added this option so you can selectively dump only the non-extra values if you want (like we want for the embed).

I made it default to `True` so the change wouldn't break any other code. 

If you want it to default to False, it should also be a pretty quick refactor; Up to you. Just let pytest point out where things broke if you change it.